cluster-autoscaler/cloudprovider/clusterapi/internal/clusterapi_manager.go

```diff
@@ -0,0 +1,111 @@
+//go:generate ./vendor-cluster-api.sh
```

Comment:
if there isn't a reason, I wouldn't use internal packages.  There is little benefit and it complicates testing.  In general in both Kube and Origin we avoid their use.

Reply:
This is temporary and something I mentioned in the overall commit message. It was this way to bridge different k8s version some time back. The goal is to move all `cloudprovider/clusterapi/internal` up a level, removing what's in `cloudprovider/clusterapi` at the moment as we no longer need the indirection. If the general consensus is we'd rather land this commit with internal removed then I can rework the PR; I had intended to follow-up immediately with this precise change. The reason for not doing so initially was to a) not break what was known working and b) to land this so that other people can land changes too.